ui/src/reducers/user.js

```diff
@@ -1,15 +1,19 @@
 import {
   USER_REQUEST,
   USER_SUCCESS,
-  USER_FAILURE
+  USER_FAILURE,
+  USER_AUTHENTICATE_SUCCESS,
+  USER_AUTHENTICATE_FAILURE
 } from '../actions/types.js';
 
 const initialState = {
   user: {
     username: '',
```

Comment:
Know it's not part of this PR, but:

```suggestion
    name: '',
```

Reply:
@damienomurchu it's the username we get back from oauth. The user's name doesn't get sent so username makes mores sense here.

Reply:
@laurafitzgerald that's a fair point :+1: